firebase-firestore/src/main/java/com/google/firebase/firestore/core/Target.java

```diff
@@ -115,6 +116,17 @@ public boolean hasLimit() {
     return endAt;
   }
 
+  /** Returns the field filters from the given list that target the given field path. */
+  private List<FieldFilter> getFieldFiltersForPath(List<Filter> filters, FieldPath path) {
```

Comment:
These "filters" are always the instance variable, right? Can the argument be dropped?

Reply:
yeah, done.